m4/pkg.m4

```diff
@@ -0,0 +1,343 @@
+# pkg.m4 - Macros to locate and utilise pkg-config.   -*- Autoconf -*-
```

Comment:
m4 added per project convention to add m4 files to source control even if exists on system.